Author: Martell Malone <martellmalone@gmail.com>

Enables Thread local storage for clang when working with mingw64.

--- clang-3.4/lib/Basic/Targets.cpp.orig  2014-01-10 22:53:00.287255800 +0000
+++ clang-3.4/lib/Basic/Targets.cpp	2014-01-10 23:21:16.717407800 +0000
@@ -3061,7 +3061,7 @@
 public:
   WindowsX86_32TargetInfo(const llvm::Triple &Triple)
       : WindowsTargetInfo<X86_32TargetInfo>(Triple) {
-    TLSSupported = false;
+    TLSSupported = true;
     WCharType = UnsignedShort;
     DoubleAlign = LongLongAlign = 64;
     DescriptionString = "e-p:32:32:32-i1:8:8-i8:8:8-i16:16:16-i32:32:32-"
@@ -3286,7 +3286,7 @@
 public:
   WindowsX86_64TargetInfo(const llvm::Triple &Triple)
       : WindowsTargetInfo<X86_64TargetInfo>(Triple) {
-    TLSSupported = false;
+    TLSSupported = true;
     WCharType = UnsignedShort;
     LongWidth = LongAlign = 32;
     DoubleAlign = LongLongAlign = 64;
